src/Shared/Recruit.Vacancies.Client/Infrastructure/Extensions/VacancyExtensions.cs

```diff
@@ -30,9 +31,9 @@ public static class VacancyExtensions
             projectedVacancy.ClosingDate = vacancy.ClosingDate.GetValueOrDefault();
             projectedVacancy.Description = vacancy.Description;
             projectedVacancy.DisabilityConfident = vacancy.DisabilityConfident;
-            projectedVacancy.EmployerContactEmail = vacancy.EmployerContactEmail;
-            projectedVacancy.EmployerContactName = vacancy.EmployerContactName;
-            projectedVacancy.EmployerContactPhone = vacancy.EmployerContactPhone;
+            projectedVacancy.EmployerContactEmail = vacancy.OwnerType == OwnerType.Employer ? vacancy.EmployerContact?.Email : vacancy.ProviderContact?.Email;
```

Comment:
I know this is WIP so don't need need to do this now.

Have a look at the FAA. We introduced Employer Contact Details for v2 vacancies. So we can project provider contact details separately and just reuse the v1 contact information.  This way we can have both an Employer Contact and Provider Contact if we wanted in the future.

https://github.com/SkillsFundingAgency/das-findapprenticeship/blob/cfed3e411ebd27dbc9c8e7c0e505ced2b315af6a/src/SFA.Apprenticeships.Web.Candidate/ViewModels/VacancySearch/ApprenticeshipVacancyDetailViewModel.cs

Reply:
This would involve a change and I thought this Live Projection is only to project what FAA wants rather than what FAA and V2 work with therefore decided this is the simplest change possible without having to change anything on FAA or the recruit client package.

Reply:
FAA is now happy with the concept of Provider contacts and Employer contacts.  We extended FAA at the time so we could do this.

Reply:
will add Provider properties 👍